doc/esmvalcore/preprocessor.rst

```diff
@@ -364,6 +364,23 @@ the ``fx_files: [sftgif]`` mask.
 
 See also :func:`esmvalcore.preprocessor.mask_landseaice`.
 
+Glaciated masking
+-----------------
+
+For masking out glaciated areas a Natural Earth shapefile is used. To mask
+glaciated areas out, ``mask_glaciated`` can be used:
+
+.. code-block:: yaml
+
+  preprocessors:
+    preproc_mask:
+      mask_glaciated:
+        mask_out: glaciated
```

Comment:
@bascrezee Why does the `mask_glaciated` function take an argument if there is only a single valid value?